package.json

```diff
@@ -17,17 +17,21 @@
 		"@zoltu/ethereum-fetch-json-rpc": "10.1.1",
 		"@zoltu/ethereum-types": "8.2.3",
 		"@zoltu/solidity-typescript-generator": "3.0.2",
+		"@zoltu/typescript-transformer-append-js-extension": "1.0.1",
 		"ganache-cli": "6.7.0",
 		"ganache-core": "2.8.0",
 		"jasmine": "3.5.0",
 		"memdown": "5.1.0",
 		"node-fetch": "2.6.0",
 		"solc": "0.5.12",
 		"ts-node": "8.4.1",
-		"typescript": "3.7.2"
+		"typescript": "3.7.2",
+		"ttypescript": "1.5.8"
+	},
+	"dependencies": {
 	},
-	"dependencies": {},
```

Comment:
These changes look like they are from my branch.  Recommend cleaning them out of here.